hls4ml/backends/fpga/passes/hgq_proxy_model.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+
+from hls4ml.backends import Backend
+from hls4ml.backends.template import FunctionCallTemplate
+from hls4ml.model.layers import Layer
+from hls4ml.model.optimizer import OptimizerPass
+from hls4ml.model.optimizer.passes.hgq_proxy_model import FixedPointQuantizer, UnaryLUT
+from hls4ml.model.types import Source
+
+
+def to_apfixed(k, b, i, RND, SAT):
```

Comment:
These should not be necessary. The standard type conversions should handle this automatically via https://github.com/fastmachinelearning/hls4ml/blob/main/hls4ml/backends/fpga/fpga_types.py#L28 and similar.

Reply:
Is there a unified type string converter (backend agnostic) I can grab from, or do I still need to do it for every different backends?

Reply:
I believe the way it works by default is via the TransformTypes (e.g. https://github.com/fastmachinelearning/hls4ml/blob/main/hls4ml/backends/vivado/passes/transform_types.py). That determines the text-based implementation for internal types. See if there's a way to use the same machinery somehow.